lib/gui/components/safe-webview.js

```diff
@@ -69,6 +85,7 @@ class SafeWebview extends react.PureComponent {
 
     // We set the version GET parameter here.
     url.searchParams.set(VERSION_PARAM, packageJSON.version);
```

Comment:
Perhaps the `VERSION_PARAM` constant should now be renamed to `ETCHER_VERSION_PARAM` ?
And I guess the comment above could be changed to `// We set the version GET parameters here.` ;)